### DISCUSSION TITLE
Page white screen

### DISCUSSION THREAD

**Discussion by qhdkunpeng** [cefsharp/CefSharp#4753] [General]
### Is there an existing issue for this?

- [x] I have searched both open/closed issues, no issue already exists.

### CefSharp Version

97.1.60

### Operating System

Windows 10

### Architecture

x64

### .Net Version

4.8

### Implementation

WinForms

### Reproduction Steps

What is the problem when Cef calls the JavaScript method and the screen turns white when switching pages?

### Expected behavior

When switching pages, Cef calls the JavaScript method with a white screen and occasionally occurs

code：
 m_chromeBrowser.ExecuteScriptAsync(funcName, JsonConvert.SerializeObject(rstJ));

### Actual behavior

When switching pages, Cef calls the JavaScript method with a white screen and occasionally occurs

### Does this problem also occur in the CEF Sample Application

Yes using WinForms command line args

**Comment by amaitland** [comment]
> What version are you using? Please only open an issue if you can reproduce the problem with version 120.1.80 or later.

Please don't open issues for old unsupported versions. Use discussions to ask for help.

> Reproduction Steps
> What is the problem when Cef calls the JavaScript method and the screen turns white when switching pages?

This isn't a set of steps to reproduce the problem.

It sounds like the render process has crashed. You can check the log file for errors.

https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#log-file

Upgrade to a support version.
